src/pages/home-page/Hero.scss

```diff
@@ -7,15 +7,15 @@
     flex-shrink: 0;
     display: flex;
     justify-content: center;
-    align-items: center; 
-   
-  }
+    align-items: center;
+
+    @media (min-width: 901px) {
+      height: 696px;}
 
```

Comment:
same in here if we use the medias for the desktop changes since  we are using mobile fisrt

Reply:
No repair needed